sympy/physics/mechanics/tests/test_kane3.py

```diff
@@ -1,4 +1,4 @@
-from sympy.core.evalf import evalf
+import sympy.core.evalf as evalf
```

Comment:
This seems to import the module instead of the function.

Reply:
yes the module is needed because of like 189.
```
    HTA                             =  evalf.N(pi / 2 - pi / 10)
```
 Before it imported just the function so that didnt work. I think it doesn't fail on the server because this test is skipped on travis